### ISSUE TITLE
Neat stuff

### ISSUE THREAD

**Issue by Thraka** [Aeirex/Eclipse-Worlds#1]
This is some pretty neat stuff you got here. I've been looking around the net, looking at all these engines built off of some original engine (I think the name was Mirage, but it was so long ago). I created SadScript and some guy used it in his 2D Zelda clone. I think it's blossomed into these games. 

Neat stuff.

**Comment by abhijeetviswa** [comment]
Well, SadScript was removed a long time ago. And yes, these Engines are based off of Mirage in one way or another.

**Comment by Aeirex** [comment]
Sadscript was never in Eclipse Origins, it was dropped after Eclipse Stable. Robin basically took Mirage Source 4 and Mirage Realms, to make Eclipse Origins many years ago. This is based off of that.

**Comment by Thraka** [comment]
Sure I know that, I didn't see it in your code base ;) But I was just commenting how neat it is to see how Mirage has evolved over time with everyone making new versions of it. Code sharing is awesome :)

I found a question on stackoverflow regarding "What is SadScript, I keep seeing it X Y Z" which took me by surprise so I decided to search researching where it has been and gone and eventually landed on the forums and then here.

SadScript was just a wrapper (as I'm sure you guys know since you're knee deep in code) that made it easier to work with the MS scripting dll. 

Cheers!
